Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.6.5-slim
```

Comment:
consider updating to be multi-stage Alpine-based docker image to reduce image size

Reply:
+1

Reply:
+1